.travis.yml

```diff
@@ -0,0 +1,7 @@
+git checlanguage: generic
+before_script:
+  - git clone https://github.com/flutter/flutter.git -b stable
+  - export PATH=`pwd`/flutter/bin:`pwd`/flutter/bin/cache/dart-sdk/bin:$PATH
+script:
+  - flutter packages get
+  - flutter test
```

Comment:
There's no building of the native parts and we should definitely check that on CI.

Reply:
It is not so obvious how to set up a pure native build. I added a ticket for iOS https://github.com/Polidea/FlutterBleLib/issues/75